File: Chapter 04/C Code/4_16.c
#include<stdio.h>
void main()
{
int a[50], f[50], n,count, i, j;

printf("Enter the size of list: ");
scanf("%d", &n);
for( i =0; i<n; i++)
{
scanf("%d", &a[i]);
f[i]= -1;
}
for( i =0; i<n; i++)
{
count=0;
if (f[i] == 1)
continue;
for( j =0; j<n; j++)
{
if (a[i]==a[j])
{
f[j]=1;
count=count+1;
}
}
printf("\nFrequency of %d is %d ", a[i], count);
}
}

File: Chapter 04/Python code/4_16.py

n=int(input("Enter the size of list: "))
a=[]
f=[]
for i in range(0,n):
    x=int(input("Enter the number: "))
    a.append(x)
    f.append(-1)
for i in range(0, n):
    count=0
    if f[i] == 1:
        continue
    for j in range(0,n):
        if a[i]==a[j]:
            f[j]=1
            count=count+1
    print("Freqency of ", a[i] , " is ", count)


